awscli/customizations/cloudfront.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from datetime import datetime
+
+from awscli.arguments import CustomArgument
+from awscli.customizations.utils import validate_mutually_exclusive_handler
+
+
+def register(event_handler):
+    """Provides a simpler --paths for ``aws cloudfront create-invalidation``"""
+
+    event_handler.register(
+        'building-argument-table.cloudfront.create-invalidation', _add_paths)
+    event_handler.register(
+        'operation-args-parsed.cloudfront.create-invalidation',
+        validate_mutually_exclusive_handler(['invalidation_batch'], ['paths']))
+
+
+def _add_paths(argument_table, **kwargs):
+    argument_table['invalidation-batch'].required = False
+    argument_table['paths'] = PathsArgument()
+
+
+class PathsArgument(CustomArgument):
+
+    def __init__(self):
+        help = """The space-separated paths to be invalidated.
+            It will automatically generate a CallerReference for you."""
+        super(PathsArgument, self).__init__('paths', nargs='+', help_text=help)
+
+    def add_to_params(self, parameters, value):
+        if value:
```

Comment:
`if value is not None` (yes, I see the nargs='+' above, but other customizations are explicit about a value being provided vs. empty value).